vector/src/main/res/layout/fragment_ftue_auth_use_case.xml

```diff
@@ -0,0 +1,167 @@
+        style="@style/Widget.Vector.TextView.Subtitle"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="16dp"
+        android:background="@drawable/bg_login_server_selector"
+        android:drawablePadding="16dp"
+        android:gravity="center_vertical"
+        android:padding="16dp"
+        android:text="@string/ftue_auth_use_case_option_two"
+        android:textColor="?vctr_content_primary"
+        app:drawableStartCompat="@drawable/ic_teams"
+        app:layout_constraintBottom_toTopOf="@id/useCaseOptionThree"
+        app:layout_constraintEnd_toEndOf="@id/useCaseGutterEnd"
+        app:layout_constraintStart_toStartOf="@id/useCaseGutterStart"
+        app:layout_constraintTop_toBottomOf="@id/useCaseOptionOne" />
+
+    <TextView
+        android:id="@+id/useCaseOptionThree"
+        style="@style/Widget.Vector.TextView.Subtitle"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="16dp"
+        android:background="@drawable/bg_login_server_selector"
+        android:drawablePadding="16dp"
+        android:gravity="center_vertical"
+        android:padding="16dp"
+        android:text="@string/ftue_auth_use_case_option_three"
+        android:textColor="?vctr_content_primary"
+        app:drawableStartCompat="@drawable/ic_communities"
+        app:layout_constraintBottom_toTopOf="@id/useCaseSkip"
+        app:layout_constraintEnd_toEndOf="@id/useCaseGutterEnd"
+        app:layout_constraintStart_toStartOf="@id/useCaseGutterStart"
+        app:layout_constraintTop_toBottomOf="@id/useCaseOptionTwo" />
+
+    <TextView
+        android:id="@+id/useCaseSkip"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:gravity="center"
+        android:clickable="true"
+        app:layout_constraintBottom_toTopOf="@id/contentFooterSpacing"
+        app:layout_constraintEnd_toEndOf="@id/useCaseGutterEnd"
+        app:layout_constraintStart_toStartOf="@id/useCaseGutterStart"
+        app:layout_constraintTop_toBottomOf="@id/useCaseOptionThree" />
+
+    <Space
+        android:id="@+id/contentFooterSpacing"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        app:layout_constraintBottom_toTopOf="@id/useCaseFooter"
+        app:layout_constraintTop_toBottomOf="@id/useCaseSkip" />
+
+    <TextView
+        android:id="@+id/useCaseFooter"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:gravity="center"
+        android:text="@string/ftue_auth_use_case_join_existing_server"
+        app:layout_constraintBottom_toTopOf="@id/useCaseConnectToServer"
+        app:layout_constraintEnd_toEndOf="@id/useCaseGutterEnd"
+        app:layout_constraintStart_toStartOf="@id/useCaseGutterStart"
+        app:layout_constraintTop_toBottomOf="@id/contentFooterSpacing" />
+
+    <TextView
```

Comment:
Better to use a `Button` here.

Reply:
47c9e75a77250ce36d6c01659ea6edea0b41a8a2 also updated the splash `I already know this` to use a button as well